plugin/pkg/scheduler/algorithm/predicates/predicates_test.go

```diff
@@ -1392,6 +1392,32 @@ func TestEBSVolumeCountConflicts(t *testing.T) {
 			},
 		},
 	}
+	deletedPVCPod := &api.Pod{
```

Comment:
if, for some reason, you end up doing another revision of this, it might make sense to rename "deletedPV" to "pvcWithDeletedPV" (or something similar), which I think would make it a bit clearer.